src/includes/performance/retrieve-transaction/dotnet.mdx

```diff
@@ -1,6 +1,6 @@
 ## Retrieve a Transaction
 
-In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry#GetSpan`. This method will return a `SentryTransaction` in case there is a running Transaction or a `Span` in case there is already a running Span, otherwise it returns `null`.
+In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry#GetSpan`. If there is a running Transaction or Span currently on the scope, this method will return a `SentryTransaction` or `Span`, otherwise it returns `null`.
```

Comment:
```suggestion
In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry#GetSpan`. If there is a running Transaction or Span currently on the scope, this method will return a `SentryTransaction` or `Span`; otherwise, it returns `null`.
```